test/lib/cache-lib/test.redis.integration.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+//var redis = require('redis');
+var assert = require('assert');
+// conditional test.
+var runTest = false;
+
+if(Number(process.env.MULTICACHE_USE_REDIS)  === 1){
+  runTest = true;
+}
```

Comment:
Lines 6 to 10 could be squashed into 1:
`+var runTest = Number(process.env.MULTICACHE_USE_REDIS) === 1;`